test/new-e2e/tests/windows/install-test/install_test.go

```diff
@@ -0,0 +1,47 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+package installscript
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2os"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2params"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/params"
+	windows "github.com/DataDog/datadog-agent/test/new-e2e/tests/windows"
+	windowsAgent "github.com/DataDog/datadog-agent/test/new-e2e/tests/windows/agent"
+
+	"testing"
+)
+
+type agentMSISuite struct {
+	e2e.Suite[e2e.VMEnv]
+}
+
+func TestMSI(t *testing.T) {
+	e2e.Run(t,
+		&agentMSISuite{},
+		e2e.EC2VMStackDef(ec2params.WithOS(ec2os.WindowsOS)),
+		params.WithStackName(fmt.Sprintf("msi-test-%v", os.Getenv("CI_PIPELINE_ID"))),
+		params.WithDevMode())
+}
+
+func (is *agentMSISuite) TestInstallAgent() {
+	vm := is.Env().VM.(*client.PulumiStackVM)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why do you need to explicitly cast to `(*client.PulumiStackVM)` ?